links.opam

```diff
@@ -18,7 +18,7 @@ depends: [
   "ocaml" {>= "4.06.0"}
   "dune" {build}
   "ppx_deriving"
-  "ppx_deriving_yojson"
+  "ppx_deriving_yojson" {> "3.1"}
```

Comment:
I'd pick `{>= "3.3"}` which is the current release. It should work with `ocaml >= 4.04`, which is fine since we require at least `4.06`.

Reply:
there is no version 3.2, so those two constraints are equivalent. I have no particular preference about expressing it with > or >=. Do you?

Reply:
I would go with the more specific constraint >= 3.3 just in case our assumption that version numbers are monotonic over time turns out to be optimistic.  (and it better documents that 3.3 is known to work without relying on context.)